hedera-node/src/main/java/com/hedera/services/context/WorkingState.java

```diff
@@ -0,0 +1,147 @@
+package com.hedera.services.context;
+
+/*-
+ * ‌
+ * Hedera Services Node
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.hedera.services.state.merkle.MerkleAccount;
+import com.hedera.services.state.merkle.MerkleBlobMeta;
+import com.hedera.services.state.merkle.MerkleDiskFs;
+import com.hedera.services.state.merkle.MerkleEntityAssociation;
+import com.hedera.services.state.merkle.MerkleEntityId;
+import com.hedera.services.state.merkle.MerkleNetworkContext;
+import com.hedera.services.state.merkle.MerkleOptionalBlob;
+import com.hedera.services.state.merkle.MerkleSchedule;
+import com.hedera.services.state.merkle.MerkleToken;
+import com.hedera.services.state.merkle.MerkleTokenRelStatus;
+import com.hedera.services.state.merkle.MerkleTopic;
+import com.swirlds.common.AddressBook;
+import com.swirlds.fcmap.FCMap;
+
+/**
+ * Manages the working state of the services. This gets updated by {@link ServicesContext} on a regular interval. The
+ * intention of this class is to avoid making repetitive calls to get the state when we know it has not yet been updated.
+ */
+public class WorkingState {
+
+	/** Working state of accounts */
+	private FCMap<MerkleEntityId, MerkleAccount> accounts = new FCMap<>();
```

Comment:
Notice we would never want to use an `FCMap` or `AddressBook` created by this class, since its role is to be a passive container of children _from_ a `ServicesState` instance. 

It seems safer to eliminate even the _possibility_ of such incorrect usage by initializing all the fields here to `null`.